docker/Dockerfile

```diff
@@ -37,16 +46,19 @@ RUN set -x && \
 
 ## librdkafka
 RUN set -x && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends --no-install-suggests \
-        g++ \
-    && \
     mkdir -p $HOME/librdkafka && \
     cd $HOME/librdkafka && \
     git clone https://github.com/edenhill/librdkafka.git . && \
     git checkout v1.0.0 && \
-    ./configure && make && make install && \
-    pecl install rdkafka && \
+    ./configure && make && make install
+
```

Comment:
Using `pecl install rdkafka-5.0.2` was not working?

Reply:
With PHP 8.x when using `pecl` it produce a PHP warning (`Invalid argument supplied for foreach() in /usr/share/pear/PEAR/...`) that break the image build.

That's why I move it to compile it manually.